assets/quick-buy.css

```diff
@@ -0,0 +1,197 @@
+  0 0 0.5rem 0.4rem rgba(var(--color-foreground), 0.3),
+  var(--popup-shadow-horizontal-offset) var(--popup-shadow-vertical-offset) var(--popup-shadow-blur-radius) rgba(var(--color-shadow), var(--popup-shadow-opacity));
+}
+
+@media screen and (min-width: 750px) {
+  .quick-buy-modal__content {
+    padding-right: 1.5rem;
+    margin-top: 10rem;
+    width: 70%;
+    padding: 0 3rem 2rem 3rem;
+  }
+}
+
+.quick-buy-modal__content img {
+  max-width: 100%;
+}
+
+.quick-buy-modal__content-info {
+  padding-right: 4.4rem;
+}
+
+.quick-buy-modal__content-info > * {
+  height: auto;
+  margin: 0 auto;
+  max-width: 100%;
+  width: 100%;
+}
+
+@media screen and (max-width: 749px) {
+  .quick-buy-modal__content-info > * {
+    max-height: 100%;
+  }
+}
+
+.quick-buy-modal__toggle {
+  background-color: rgb(var(--color-background));
+  border: 0.1rem solid rgba(var(--color-foreground), 0.1);
+  border-radius: 50%;
+  color: rgba(var(--color-foreground), 0.55);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  position: sticky;
+  padding: 1.2rem;
+  z-index: 5;
+  top: 1.5rem;
+  width: 4rem;
+  margin: 0 0 0 auto;
+  margin-bottom: -0.8rem;
+}
+
+.quick-buy-modal__toggle:hover {
+  color: rgba(var(--color-foreground), 0.75);
+}
+
+.quick-buy-modal__toggle .icon {
+  height: auto;
+  margin: 0;
+  width: 2.2rem;
+}
+
+.quick-buy-modal__content {
+  max-width: var(--page-width);
+}
+
+quickbuy-modal .quick-buy-modal__content-info {
+  padding-right: 0;
+}
+
+quickbuy-modal .product:not(.featured-product) .product__view-details {
+  display: block;
+}
+
+quickbuy-modal .product__media-toggle,
+quickbuy-modal .skip-to-content-link,
+quickbuy-modal .product__media-icon,
+quickbuy-modal .slider-buttons,
+quickbuy-modal .thumbnail-list,
+quickbuy-modal .product-popup-modal__opener,
+quickbuy-modal .product__modal-opener:not(.product__modal-opener--image),
+quickbuy-modal .product__media-item:not(:first-child),
+quickbuy-modal pickup-availability,
+quickbuy-modal .product__description,
+quickbuy-modal .product__accordion,
+quickbuy-modal .share-button,
+quickbuy-modal .slider-button,
+quickbuy-modal product-modal {
+  display: none !important;
+}
+
+quickbuy-modal .product__media-list .deferred-media {
+  display: block;
+  width: 100%;
+}
+
+@media screen and (max-width: 749px) {
+  quickbuy-modal .product__media-wrapper slider-component:not(.thumbnail-slider--no-slide) {
+    margin-right: 0;
+  }
+
+  quickbuy-modal .slider.product__media-list .product__media-item:first-of-type {
+    padding-right: 1.5rem;
+  }
+}
+
+@media screen and (min-width: 750px) {
+  .quick-buy-modal__toggle {
+    margin: 0 0 0 auto;
+  }
+
+  quickbuy-modal .product--small:not(.product--no-media) .product__media-wrapper,
+  quickbuy-modal .product--medium:not(.product--no-media) .product__media-wrapper,
+  quickbuy-modal .product--large:not(.product--no-media) .product__media-wrapper {
```

Comment:
It might be interesting for the modal to have different sizing options or follow the product page settings. Though that would too "magical". But I could see it being useful when having landscape medias.

Reply:
Thanks for the feedback Ludo, that's an interesting thought! But I would park the modal explorations for now, our focus for mvp is the quick add per brief. cc: @LucasLacerdaUX